src/main/java/seedu/address/model/item/Item.java

```diff
@@ -0,0 +1,79 @@
+package seedu.address.model.item;
+
+import java.util.Optional;
+import java.util.TreeSet;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+/**
+ * A stub class for the items.
+ */
+public class Item {
+    private String description;
+    private Optional<Task> task;
+    private Optional<Event> event;
+    private Optional<Reminder> reminder;
+    private TreeSet<String> tags;
```

Comment:
Are we using Strings for tags? Maybe we should use tag objects instead so that we can group and sort later on?

Reply:
Ok! Depends on how you would like to implement it then!